Doc/whatsnew/3.14.rst

```diff
@@ -929,6 +929,62 @@ sys
 * On FreeBSD, :data:`sys.platform` doesn't contain the major version anymore.
   It is always ``'freebsd'``, instead of ``'freebsd13'`` or ``'freebsd14'``.
 
+* Schedule a change for availability of :data:`sys.abiflags` on Windows. A
+  :exc:`DeprecationWarning` will be emitted if the :data:`sys.abiflags` member
+  is accessed on Windows before Python 3.16.
+  (Contributed by Xuehai Pan in :gh:`131717`.)
+
+.. _whatsnew314-sys-abiflags-change:
+
+sys.abiflags
+------------
+
+A :exc:`DeprecationWarning` will be emitted if the :data:`sys.abiflags` member
+is accessed on Windows before Python 3.16.
+For example:
+
+.. code-block:: python
+
+   >>> import sys
+   >>> getattr(sys, 'abiflags', None)  # on Windows
+   <python-input-1>:1: DeprecationWarning: sys.abiflags will be set to a meaningful value on all platforms ...
+   >>> hasattr(sys, 'abiflags')  # on Windows
+   <python-input-2>:1: DeprecationWarning: sys.abiflags will be set to a meaningful value on all platforms ...
+   False
+
+To suppress this warning, use the :mod:`warnings` module:
+
+.. code-block:: python
+
+   import warnings
+
+   with warnings.catch_warnings():
+       # ignore DeprecationWarning on sys.abiflags change on Windows
+       warnings.simplefilter('ignore', DeprecationWarning)
+       abiflags = getattr(sys, 'abiflags', '')
```

Comment:
I don't think the example is needed. But it doesn't hurt (having a link to the warnings module should be sufficient IMO)

Reply:
I add this by following the suggestion in https://github.com/python/cpython/pull/131717#discussion_r2013007046.

Reply:
Ah I see. Yes, ok though I think it's a bit too pedantic (the section becomes too long IMO).